src/converters/lintConfigs/rules/ruleConverters.ts

```diff
@@ -518,12 +540,23 @@ export const ruleConverters = new Map([
     ["react-tsx-curly-spacing", convertReactTsxCurlySpacing],
     ["relative-url-prefix", convertRelativeUrlPrefix],
     ["restrict-plus-operands", convertRestrictPlusOperands],
+    ["rxjs-ban-observables", convertBanObservables],
+    ["rxjs-ban-operators", convertBanOperators],
+    ["rxjs-finnish", convertFinnish],
```

Comment:
This got me confused for a bit: there's both `finnish` and `no-finnish` as separate rules. Got it!